src/lib/annotations/AnnotationThread.js

```diff
@@ -551,6 +568,27 @@ class AnnotationThread extends EventEmitter {
     deleteAnnotationWithID(data) {
         this.deleteAnnotation(data.annotationID);
     }
+
+    /**
+     * Inserts user initials and updates point annotation icon with a
+     * color pseudo-unique to the user
+     *
+     * @private
+     * @param {Object} annotation - first annotation in the thread
+     * @return {void}
+     */
+    fillPointAnnotationIcon(annotation) {
+        // Update point annotation icon with user initials
+        const userName = annotation.user.name;
+        const userInitials = userName
+            .split(' ')
+            .map((n) => n[0])
+            .join('');
+        this.element.innerHTML = userInitials;
```

Comment:
Answered my own question. Does this work with missing username? Does this cap at two initials?

Reply:
It doesn't cap at 2 initials. I'm not entirely sure how we'd deal w/ people with a lot of names in their full name. Maybe cap it at 2 to include the most common case of first, middle and last? I'll confirm this  

All users are required to have a name so we wouldn't run into a null case here but we do have the workaround that we've been seeing where a user can have a space for the name? We decided to allow a blank circle in those cases.

Reply:
Going to only generate initials from the user's first and last name. See the tests here
https://github.com/box/box-content-preview/pull/380/commits/5255bb2c03e996f915f8dce911b7dd0db1882fba